lmdeploy/pytorch/config.py

```diff
@@ -46,7 +46,8 @@ class ModelConfig:
 
     hidden_size: int
     num_layers: int
-    num_heads: int
+    num_attention_heads: int
```

Comment:
may update the supported model names for pytorch engine in 
https://github.com/InternLM/lmdeploy/blob/0f324a4cf7dabcf5de742e0ee7031c699ae5195f/lmdeploy/cli/cli.py#L108-L111

and docs like
- https://github.com/InternLM/lmdeploy/blob/main/docs/en/supported_models/supported_models.md#models-supported-by-pytorch
- https://github.com/InternLM/lmdeploy/blob/main/README.md#supported-models